src/ZEO/ClientStorage.py

```diff
@@ -53,6 +53,9 @@
 
 logger = logging.getLogger(__name__)
 
+# max signed 64-bit value ~ infinity :) Signed cuz LBTree and TimeStamp
+m64 = b'\x7f\xff\xff\xff\xff\xff\xff\xff'
```

Comment:
Well, z64 is lower case, and this is related.  I expect this will eventually migrate to ZODB.utils.

What tests need the value?